xml/art_sle_ha_install_quick.xml

```diff
@@ -788,6 +793,98 @@ softdog                16384  1</screen>
        <keycap function="control"/><keycap>C</keycap>
       </keycombo>.
    </para>
+   </sect2>
+
+   <sect2 xml:id="sec-ha-inst-quick-test-with-cluster-script">
+    <title>Testing with the &test.script; Script</title>
+    <para>
+     The command &cmd.test.script; is a tool to run standardized tests
+     for a cluster.
+     It triggers cluster failures and verify configuration to find
+     problems. Before you use your cluster in production, it is
+     recommended to use this script to make sure everything works as expected.
+    </para>
+    <para>
+     The script supports the following checks:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <formalpara>
+       <title>Environment Check <option>-e</option>/<option>--env-check</option></title>
+       <para>
+        Check if hostname is resolvable, time service, watchdog, and
+       firewall.
+       </para>
+      </formalpara>
+     </listitem>
+     <listitem>
+      <formalpara>
+       <title>Cluster State Check <option>-c</option>/<option>--cluster-check</option></title>
+       <para>
+        Check cluster service, &stonith;/fence configuration, nodes,
+        and resources.
+       </para>
+      </formalpara>
+     </listitem>
+     <listitem>
+      <formalpara>
+       <title>Split Brain Check <option>--split-brain-iptables</option></title>
+       <para>
+        Simulates split brain scenario by blocking the &corosync; port.
+        Checks wheather one node can be fenced as expected.
+       </para>
+      </formalpara>
+     </listitem>
+     <listitem>
+      <formalpara>
+       <title>Killing Test Cases <option>-kill-sbd</option>/<option>-kill-corosync</option>/<option>-kill-pacemakerd</option></title>
```

Comment:
Probably too late to change it in the software for SLE HA 15 SP2, but 'killing' is a rather unfortunate term to use in this regard. At least in the documentation, I would change it to something like 
```Stops the daemons for SBD, &corosync;, and &pace;.```

Reply:
@liangxin1300 Xin, is it too late to change the options in the command? Could we use `--stop-*` instead?  I can also open an issue in your repo, if it's possible. Just asking. :wink:

Nevertheless, I've changed the line as Tanja advised.

Reply:
Ok, clarified difference between "stopping" and "killing" of resources in an email. Thanks to Lars. Setting question as resolved.